src/main/java/com/gmail/nossr50/util/blockmeta/chunkmeta/PrimitiveChunkStore.java

```diff
@@ -94,14 +105,18 @@ private void set(int x, int y, int z, boolean val) {
         int point = (y * PAGE_SIZE) + (x >> 1);
         int bit = z + (((x & 1) == 1) ? 16 : 0);
         int source;
+        // An explanation of the operation of shift is on lines 26 to 42
```

Comment:
Line numbers change when the file is edited, I'd prefer a comment that didn't need to be regularly updated